release_tools/changelog.py

```diff
@@ -208,16 +211,36 @@ def create_changelog_entry_content(title, category, author=None, issue=None,
     else:
         content = stream
 
+    if not content:
+        msg = "Aborting due to empty entry content"
+        raise click.ClickException(msg)
+
+    try:
+        data = yaml.safe_load(content)
+    except yaml.YAMLError as exc:
+        pm = exc.problem_mark
+        msg = "File has an issue on line {} at position {}".format(pm.line, pm.column)
+        raise click.ClickException(msg)
+
+    if data['author'] and re.search(REGEX, data['author']):
+        pass
+    else:
+        msg = "Author field should be of the format `John Smith <jsmith@example.com>`"
+        raise click.ClickException(msg)
```

Comment:
I don't think we need to check the format of the author. There might be people using nicknames only. Remember that the idea is that this tool can work for everybody or any python project.